src/2D-Top-down-shooter.cpp

```diff
@@ -1,13 +1,13 @@
 #include <GL/freeglut.h>
 #include "GameObject.h"
-#include <iostream>
-
-GameObject gameObject(0.5, 0.5);
+#include "MenuObject.h"
 
 void displayMe(void) {
-
+	GameObject gameObject(0.5, 0.5);
```

Comment:
The initialization should be outside of the function, because we are using the gameObject in other fucntions too.